moped-editor/src/views/projects/projectView/ProjectActivityLog.js

```diff
@@ -259,131 +266,109 @@ const ProjectActivityLog = () => {
                       </TableCell>
                       <TableCell
                         align="left"
-                        width="5%"
-                        className={classes.tableCell}
-                      >
-                        {getOperationName(
-                          change.operation_type,
-                          change.record_type // todo: could this be better
-                        )}
-                      </TableCell>
-                      <TableCell
-                        align="left"
-                        // if the details link comes back, this goes back to 70%
-                        width="75%"
+                        width="80%"
                         className={classes.tableCell}
                       >
-                        <Box display="flex" p={0}>
-                          <Box p={0}>
-                            <Icon>
-                              {getChangeIcon(
-                                change.operation_type,
-                                change.record_type
-                              )}
-                            </Icon>
-                          </Box>
-                          <Box p={0} flexGrow={1}>
-                            <Grid
-                              container
-                              className={classes.updateGridEntries}
-                            >
-                              {Array.isArray(change.description) &&
-                                change.description.length === 0 && (
-                                  <Grid
-                                    item
-                                    className={classes.tableChangeItem}
-                                  >
-                                    <span className={classes.boldText}>
-                                      {getCreationLabel(
-                                        change,
-                                        userList,
-                                        phaseList
-                                      )}
-                                    </span>
-                                  </Grid>
+                        {["moped_project", "moped_proj_tags"].includes(change.record_type) ? (
+                          selectEntryComponent(change)
```

Comment:
this is temporary, the array of table names. Once everything is updated and I remove the old way of showing events, we won't have to use this wonky thing.